concert/app/src/functions/on_event_stage_changed/index.py

```diff
@@ -19,4 +32,26 @@ def handler(event, context):
         },
     )
 
+    stage_record_update_response = update_stage_record(message)
```

Comment:
What do you think about putting the DynamoDB update first, so that the SNS message isn't sent until the DB is updated properly? If the DB update fails, it'll keep retrying and message won't be sent until success.

Reply:
Makes a lot of sense. Done.